starter-code/bin/seeds.js

```diff
@@ -0,0 +1,21 @@
+
+
+
+const MovieModel = require('../models/movie.model');
+require('../app');
+
+// let seedData = [{name: 'Tom Cruise',occupation: 'actor', catchPhrase: 'Nothing is imposible'},{name: 'Beyonce',occupation: 'singer', catchPhrase: 'I\'m a Single lady'},
```

Comment:
for readability, please remember to put the objects in an array in separate lines.